tools/accuracy_checker/accuracy_checker/annotation_converters/im2latex.py

```diff
@@ -40,9 +40,7 @@ def read_vocab(vocab_path):
     elif vocab_path.suffix == '.json':
         with open(vocab_path, "r") as f:
             vocab_dict = json.load(f)
-            for k, v in vocab_dict['id2sign'].items():
-                del vocab_dict['id2sign'][k]
-                vocab_dict['id2sign'][int(k)] = v
+            vocab_dict['id2sign'] = {int(k): v for k, v in vocab_dict['id2sign'].items()}
```

Comment:
do you can guarantee that keys are always can be casted to the int?

Reply:
you still modify vocab during iteration
```suggestion
            id2sign = {int(k): v for k, v in vocab_dict['id2sign'].items()}
            vocab_dict['id2sign'] = id2sign
```
or provide json reading scheme which will cast it automatically